x/monitoringc/keeper/msg_server_create_client.go

```diff
@@ -72,6 +72,10 @@ func (k msgServer) CreateClient(goCtx context.Context, msg *types.MsgCreateClien
 		ClientID: clientID,
 	})
 
+	// set chain to be IBC connected
+	chain.IbcConnected = true
+	k.launchKeeper.SetChain(ctx, chain)
```

Comment:
I would prefer having `k.launchKeeper.EnableMonitoringConnection(ctx, chainID)` or `MonitoringConnected(...)` to prevent the module to directly write into the store to keep some kind of encapsulation

There would be a duplicated read in the store for the `chainID` but it's in my opinion not significant since establishing a connection to a testnet is already an extensive operation (with relayers etc...)